openmls/src/group/tests/mod.rs

```diff
@@ -1,5 +1,6 @@
 //! Unit tests for the core group
 
+#[cfg(any(feature = "test-utils", test))]
```

Comment:
These two `cfg` guards shouldn't be necessary.

Reply:
Now that the module is guarded, I could reduce the scope.